docs/CONFIG-PROPERTIES.md

```diff
@@ -31,8 +31,10 @@
 | debug.enable.vga | boolean | false | allow VGA console on device |
 | debug.enable.ssh | authorized ssh key | empty string(ssh disabled) | allow ssh to EVE |
 | debug.enable.console | boolean | false | allow console access to EVE (reboot required to disable) |
-| debug.default.loglevel | string | info | min level saved in files on device |
-| debug.default.remote.loglevel | string | warning | min level sent to controller |
+| debug.default.loglevel | string | info | min level saved in files on device. Used logrus log levels as described here ["https://pkg.go.dev/github.com/sirupsen/logrus"]: panic, fatal, error, warning, info, debug and trace.
+| debug.syslog.loglevel | string | info | min level of the syslog messages saved in files on device. System default loglevel string representation should be used as described here ["https://man7.org/linux/man-pages/man3/syslog.3.html"]: emerg, alert, crit, err, warning, notice, info, debug. |
+| debug.kernel.loglevel | string | info | min level of the kernel messages saved in files on device. System default loglevel string representation should be used as described here ["https://man7.org/linux/man-pages/man3/syslog.3.html"]: emerg, alert, crit, err, warning, notice, info, debug. |
```

Comment:
I see "warn" in the logs; I was assuming that was coming from kernel or syslog since logrus uses "warning". Is that correct? If so the above is not correct.

Reply:
This is how the syslog defines them:
```
       LOG_ERR        error conditions
       LOG_WARNING    warning conditions
```

this is how kernel defines them:

```
       KERN_ERR               3        Error conditions
       KERN_WARNING           4        Warning conditions
```

Reply:
I do not see "warn", only warning for all the 3 sources of logs, so this PR is correct:
```
[2024-03-18 15:25:54] <warning>  kernel                                        FAT-fs (sda4): utf8 is not a recommended IO charset for FAT filesystems, filesystem will be case sensitive!
[2024-03-18 15:26:05] <warning>  dhcpcd                                        eth0: no IPv6 Routers available
[2024-03-18 15:26:39] <warning>  volumemgr:volumemgr.Run(volumemgr.go:581)     Subject to watchdog. Waiting for verifierRestarted
```

Reply:
I know I've seen "err" and "error" in the logs and I'm pretty sure I've seen "warn" and "warning" as well.
But if it is defined as WARNING the doc is correct.